### PR TITLE
Fixes RouteInfoInterface type hint in handleRoute() method

### PR DESCRIPTION
**Pull request by samsonasik**
as the `RouteInfoInterface` type hint make `RuntimeException` never executed. It fixes now ;)

### CODE REVIEW

File: src/App.php [hunk 1]
```diff
@@ -133,15 +133,13 @@ public function run($request = null, $response = null)
     /**
      * Handle Route.
      *
-     * @param RouteInfoInterface $routeInfo
+     * @param $routeInfo
```
Comment (EmanueleMinotto) [inline src/App.php]: add `mixed`

**Comment by samsonasik** [comment]
it need rebase if #140 will be merged ;)

File: src/App.php (hunk 1 above)
Comment (samsonasik) [inline src/App.php]: done ;)

**Comment by gianarb** [comment]
Thanks

**Pull request merged**